### PR TITLE
Add nav to simulation page, tweak lab landing page

### PR DESCRIPTION
**Pull request by b30wulffz**

### CODE REVIEW

File: templates/partials/simulation-header.handlebars [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="backdrop vlabs-hidden">
+        <div class="nav-menu ">
+          {{> nav_menu_items }}
+        </div>
+      </div>
+    </div>
+
+      <a href="http://vlab.co.in" class="navbar-brand">
```
Comment (raj-vlabs) [inline templates/partials/simulation-header.handlebars:14]: Please use https link

**Review by raj-vlabs — APPROVED** [review]
Please fix the comment about using the https link and merge.

File: templates/assets/css/vlabs-style.css [hunk 2]
```diff
@@ -367,4 +366,8 @@ iframe {
 
 .simulation-header .dropdown .vlabs-hidden {
   display: none;
+}
+
+.f-1{
+  flex:1;
```
Comment (raj-vlabs) [inline templates/assets/css/vlabs-style.css:372]: this class name is very unintuitive. There's already a bootstrap class flex-grow-1 for the same purpose, please use that

File: templates/partials/footer.handlebars [hunk 3]
```diff
@@ -2,9 +2,9 @@
   <div class="row px-5 mx-0">
     <div class="col d-flex flex-column">
       <span class="font-weight-bold vlabs-footer-sect-name pb-2 mb-3">Community Links</span>
-      <a href="http://www.sakshat.ac.in/" class="text-light">Sakshat Portal</a>
-      <a href="http://outreach.vlabs.ac.in/" class="text-light">Outreach Portal</a>
-      <a href="http://vlab.co.in/faq" class="text-light">FAQ: Virtual Labs</a>
+      <a href="https://www.sakshat.ac.in/" class="text-light">Sakshat Portal</a>
+      <a href="https://outreach.vlabs.ac.in/" class="text-light">Outreach Portal</a>
```
Comment (raj-vlabs) [inline templates/partials/footer.handlebars:6]: Please use http for outreach. Unfortunately outreach doesn't support https.

**Review by raj-vlabs — APPROVED** [review]
Please resolve the 2 comments and merge.

File: templates/assets/css/vlabs-style.css (hunk 2 above)
Comment (b30wulffz) [reply templates/assets/css/vlabs-style.css:372]: I tried with flex-grow, but it was not giving desired result. Hence I updated the class name to expand-1 for better readability.

**Review by raj-vlabs — APPROVED** [review]

**Pull request merged**